SU2_CFD/src/solvers/CNSSolver.cpp

```diff
@@ -1077,3 +1476,397 @@ void CNSSolver::SetTauWall_WF(CGeometry *geometry, CSolver **solver_container, C
+                Vel[iDim]    = nodes->GetVelocity(Point_Normal,iDim);
+              P_Normal       = nodes->GetPressure(Point_Normal);
+              T_Normal       = nodes->GetTemperature(Point_Normal);
+
+              /*--- Compute the wall-parallel velocity at first point off the wall ---*/
+
+              VelNormal = 0.0;
+              for (auto iDim = 0u; iDim < nDim; iDim++)
+                VelNormal += Vel[iDim] * UnitNormal[iDim];
+              for (auto iDim = 0u; iDim < nDim; iDim++)
+                VelTang[iDim] = Vel[iDim] - VelNormal*UnitNormal[iDim];
+
+              VelTangMod = 0.0;
+              for (auto iDim = 0u; iDim < nDim; iDim++)
+                VelTangMod += VelTang[iDim]*VelTang[iDim];
+              VelTangMod = sqrt(VelTangMod);
+
+              /*--- Compute normal distance of the interior point from the wall ---*/
+
+              for (auto iDim = 0u; iDim < nDim; iDim++)
+                WallDist[iDim] = (Coord[iDim] - Coord_Normal[iDim]);
+
+              WallDistMod = 0.0;
+              for (auto iDim = 0u; iDim < nDim; iDim++)
+                WallDistMod += WallDist[iDim]*WallDist[iDim];
+              WallDistMod = sqrt(WallDistMod);
+
+              /*--- Compute the wall temperature using the Crocco-Buseman equation ---*/
+
+              //T_Wall = T_Normal * (1.0 + 0.5*Gamma_Minus_One*Recovery*M_Normal*M_Normal);
+              T_Wall = T_Normal + Recovery*pow(VelTangMod,2.0)/(2.0*Cp);
+
+              /*--- Extrapolate the pressure from the interior & compute the
+               wall density using the equation of state ---*/
+
+              P_Wall       = P_Normal;
+              Density_Wall = P_Wall/(Gas_Constant*T_Wall);
+              Lam_Visc_Wall = nodes->GetLaminarViscosity(iPoint);
+
+              /*--- Friction velocity and u+ ---*/
+
+              U_Tau  = sqrt(Tau_Wall / Density_Wall);
+              U_Plus = VelTangMod/U_Tau;
+
+              Gam  = Recovery*U_Tau*U_Tau/(2.0*Cp*T_Wall);
+              Beta = 0.0; // For adiabatic flows only
+              Q    = sqrt(Beta*Beta + 4.0*Gam);
+              Phi  = asin(-1.0*Beta/Q);
+
+              /*--- Y+ defined by White & Christoph (compressibility and heat transfer) ---*/
+
+              Y_Plus_White = exp((kappa/sqrt(Gam))*(asin((2.0*Gam*U_Plus - Beta)/Q) - Phi))*exp(-1.0*kappa*B);
+
+              /*--- If the Y+ defined by White & Christoph is too high so the eddy viscosity.
+                Disable the wall function calculation on this point and do not set the eddy viscosity
+                at the first point off the wall. ---*/
+
+              if (Y_Plus_White > 1e4){
+                cout << "WARNING: Y+ is too high (>1e4). The muT computation at the 1st point off the wall is disable." << endl;
+                cout << rank << " " << iPoint;
+//                for (auto iDim = 0u; iDim < nDim; iDim++)
+//                  cout << " " << Coord[iDim];
+                cout << endl;
```

Comment:
disable'd' in the sentence
Add `"Rank: "` and `"iPoint: "` otherwise it will be quite confusing, or integrate it in sentence above
Remove commented debug output.